e2e/du_test.go

```diff
@@ -155,7 +155,8 @@ func TestDiskUsageMissingObject(t *testing.T) {
 	result.Assert(t, icmd.Success)
 
 	assertLines(t, result.Stdout(), map[int]compareFunc{
-		0: suffix(` + 0 bytes in 0 objects: s3://%v/non-existent-file [Total]`, bucket),
-		1: equals(""),
+		0: equals("-ERR list: no object found"),
+		1: suffix(` + 0 bytes in 0 objects: s3://%v/non-existent-file [Total]`, bucket),
```

Comment:
looks bad. we need to handle this case better. both `ERR` and `0 bytes in 0 objects` are not user-friendly outputs.

Reply:
agreed. can you put TODO label for this test so that we can fix it in following PRs.